lib/src/signed_video_h26x_nalu_list.c

```diff
@@ -428,8 +429,12 @@ h26x_nalu_list_remove_missing_items(h26x_nalu_list_t *list)
       item->validation_status = 'U';
       found_decoded_sei = true;
     }
-    if (item->validation_status == 'P') found_first_pending_nalu = true;
+    if (item->validation_status == 'N') {
+      // Reset validation status to 'P' for the next validation.
+      item->validation_status = 'P';
```

Comment:
Is this really a good and proper action?
First of all, the function name says that only missing items are removed, but in fact all invalid items have their validation status changed.
Second, what will happen if we have `IPPPSIPPSI` and the first `P` of the first GOP is modified?